docs/products/kafka/kafka-connect/howto/clickhouse-sink-connector.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Create a ClickHouse sink connector for Aiven for Apache Kafka®
+sidebar_label: ClickHouse sink connector
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+
+The ClickHouse sink connector delivers data from Apache Kafka® topics to a ClickHouse database for efficient querying and analysis.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following:
+
+- An [Aiven for Apache Kafka® service](https://docs.aiven.io/docs/products/kafka/kafka-connect/howto/enable-connect)
+  with Apache Kafka Connect enabled or a
+  [dedicated Aiven for Apache Kafka Connect® service](https://docs.aiven.io/docs/products/kafka/kafka-connect/get-started#apache_kafka_connect_dedicated_cluster).
+- Access to a ClickHouse service (either Aiven for ClickHouse or an external instance),
+  including:
+  - Hostname, port, and credentials for the ClickHouse service.
+  - A pre-created target database and table.
+
+## Limitations
+
+The ClickHouse sink connector has the following limitations related to data consistency
+and exactly-once delivery:
+
+1. **No exactly-once delivery after restore**: The connector does not guarantee
```

Comment:
@harshini-rangaswamy Some feedback from @AnatolyPopov :
-  It's maybe not clear which service is restored/powered off/forked, we should mention that it's the ClickHouse one
- We should mention that at-least-once delivery is guaranteed

Reply:
Thanks, @davide-armand, Addressed this.